File: mpp-proiect-repository-cristinahendre/lab8/Java/lab6/modul/NetworkModule/src/main/java/dto/DonatorDTO.java
package dto;


import java.io.Serializable;


public class DonatorDTO implements Serializable{
    private String nume;
    private String prenume;
    private String adresa;

    @Override
    public String toString() {
        return "DonatorDTO{" +
                "nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                ", adresa='" + adresa + '\'' +
                ", tel=" + tel +
                '}';
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public String getAdresa() {
        return adresa;
    }

    public void setAdresa(String adresa) {
        this.adresa = adresa;
    }

    public long getTel() {
        return tel;
    }

    public void setTel(long tel) {
        this.tel = tel;
    }

    private long tel;

    public DonatorDTO(String nume, String prenume, String adresa, long tel) {
        this.nume = nume;
        this.prenume = prenume;
        this.adresa = adresa;
        this.tel = tel;
    }

    public DonatorDTO(String nume){
        this.nume=nume;
    }
}

File: mpp-proiect-repository-cristinahendre/lab6(web)/CSharp/Lab5/Network/DonatorDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Network
{
    [Serializable]
    public class DonatorDTO
    {
        string nume;
        string pre;
        string ad;
        long tel;

        public DonatorDTO(string n, string p, string a, long t)
        {
            this.nume = n;
            this.pre = p;
            this.ad = a;
            this.tel = t;
        }

        public DonatorDTO(string nume)
        {
            this.nume = nume;
        }

        public string Nume { get => nume; set => nume = value; }
        public string Pre { get => pre; set => pre = value; }
        public string Adresa { get => ad; set => ad = value; }
        public long Tel { get => tel; set => tel = value; }

        public override string ToString()
        {
            return "Nume = " + this.nume;
        }
    }
}
